### PR TITLE
win or lose final update

### PR DESCRIPTION
**Pull request by pnaatz**
the player being able to move glitch is specific to unity

### CODE REVIEW

File: Assets/Scripts/WinOrLoseScript.cs [hunk 1]
```diff
@@ -225,29 +223,9 @@ public void MoveHere(int podiumNumber, string playerName)
...
-            //setup for next incrementation
-            incrementation++;
-        }
-
-        return incrementation;
-    }
-
     IEnumerator Reveal()
     {
+        Debug.Log("revealing");
```
Comment (jcorm714) [inline Assets/Scripts/WinOrLoseScript.cs:228]: Do we need this log

File: Assets/Scripts/WinOrLoseScript.cs [hunk 2]
```diff
@@ -350,7 +330,12 @@ private List<int> GenerateBigestLosersList()
     {
         if(losers.Count == 0)
         {
-            return new List<int>(new int[] { 0, 1, 2, 3 });
+            List<int> fullList = new List<int>();
```
Comment (jcorm714) [inline Assets/Scripts/WinOrLoseScript.cs:333]: Add a comment to explain this

**Review by jcorm714 — CHANGES_REQUESTED** [review]

File: Assets/Scripts/WinOrLoseScript.cs (hunk 1 above)
Comment (pnaatz) [reply Assets/Scripts/WinOrLoseScript.cs:228]: completed

File: Assets/Scripts/WinOrLoseScript.cs (hunk 2 above)
Comment (pnaatz) [reply Assets/Scripts/WinOrLoseScript.cs:333]: commented

**Review by jcorm714 — APPROVED** [review]

**Pull request merged**